src/ServiceControlInstaller.Engine/Instances/ServiceControlAuditNewInstance.cs

```diff
@@ -0,0 +1,96 @@
+﻿namespace ServiceControlInstaller.Engine.Instances
+{
+    using System;
+    using System.IO;
+    using System.Linq;
+    using System.Reflection;
+    using System.Xml;
+    using System.Xml.Serialization;
+    using Configuration.ServiceControl;
+    using FileSystem;
+    using Queues;
+    using Services;
+    using Validation;
+
+    public class ServiceControlAuditNewInstance : ServiceControlInstallableBase, IServiceControlAuditInstance
+    {
+        public static ServiceControlAuditNewInstance CreateWithDefaultPersistence()
+        {
+            return CreateWithDefaultPersistence(Assembly.GetExecutingAssembly().Location);
+        }
+
+        public static ServiceControlAuditNewInstance CreateWithDefaultPersistence(string deploymentCachePath)
+        {
+            var zipInfo = ServiceControlAuditZipInfo.Find(deploymentCachePath);
+            var persistenceManifest = ServiceControlAuditPersisters.LoadAllManifests(zipInfo.FilePath)
+                .Single(manifest => manifest.Name == "RavenDb5");
```

Comment:
We have two "default" persistences. One for existing instances that don't have a persistence configured (Raven3.5) and one for brand new instances (Raven5). This info is scattered around in a few different locations. Maybe we can consolidate that somehow.